qa/standalone/osd/divergent-priors.sh

```diff
@@ -56,8 +56,8 @@ function run() {
 function TEST_divergent() {
     local dir=$1
 
+    local dummyfile=$(file_with_random_data)
     # something that is always there
-    local dummyfile='/etc/fstab'
     local dummyfile2='/etc/resolv.conf'
```

Comment:
I'm wondering why `/etc/resolv.conf` isn't subjected to the same `file_with_random_data` treatment.  Can `/etc/resolv.conf` be assumed to always be there, unlike `/etc/fstab`?

Reply:
I too was debating whether to change that. But I did not have the pretext to do that: I do not have any reason to believe that resolve.conf may be missing.
But as you have noticed that too - I think I will re-push with this change, too.